llvm/unittests/Analysis/CMakeLists.txt

```diff
@@ -80,5 +80,11 @@ if(NOT WIN32)
   export_executable_symbols_for_plugins(AnalysisTests)
 endif()
 
+include(CheckCXXSymbolExists)
+check_cxx_symbol_exists(logf128 math.h HAS_LOGF128)
+if(HAS_LOGF128)
+  target_compile_definitions(AnalysisTests PRIVATE HAS_LOGF128)
```

Comment:
Why is this needed?

It looks like nothing in AnalysisTests reads this
If it was needed in tests, shouldn't the tests include llvm-config.h and look at LLVM_HAS_LOGF128 instead anyways?

(Originally asked at https://github.com/llvm/llvm-project/commit/088aa81a545421933254f19cd3c8914a0373b493#r141559244 but here is probably better.)